app/Http/Controllers/API/Auth/LoginProxy.php

```diff
@@ -28,13 +38,22 @@ public function __construct(Application $app, User $user)
 
     /**
      * Attempt to create an access token using user credentials
-     *
-     * @param string $email
-     * @param string $password
+     * @param $email
+     * @param $password
+     * @return \Illuminate\Contracts\Foundation\Application|ResponseFactory|JsonResponse|Response|\Symfony\Component\HttpFoundation\Response
      */
     public function attemptLogin($email, $password)
     {
-        $user = User::where('email', $email)->first();
+        // fetch the users with the email
+        $user = User::where('email', $email)
```

Comment:
blocker for people logging in who shouldn't be allowed to when they have no enabled traders.
They can still have an account if all their traders are disabled, but can't log in

Reply:
Neat